### PR TITLE
GameDB: upscaling fixes and documentation  updates

### PR DESCRIPTION
**Pull request by Mrlinkwii**
Adds upscaling fixes for Tim Burton's The Nightmare Before Christmas and fix missing documentation 

### Suggested Testing Steps

 check said game and said documentation

### CODE REVIEW

**Review by JordanTheToaster — APPROVED** [review]
Fixes work correctly.

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -21937,6 +21940,12 @@ SLPM-61051:
 SLPM-61087:
   name: "Fullmetal Alchemist 2 - Curse of the Crimson Elixir [Trial]"
   region: "NTSC-J"
+SLPM-61090:
+  name: "Tim Burton's The Nightmare Before Christmas [Trial]"
+  region: "NTSC-J"
+  gsHWFixes:
+    wildArmsHack: 1 # Fixes bluryness.
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:21947]:
```suggestion
    wildArmsHack: 1 # Fixes blurriness .
```

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -26358,6 +26367,9 @@ SLPM-65739:
   name: "Nightmare Before Christmas"
   region: "NTSC-J"
   compat: 5
+  gsHWFixes:
+    wildArmsHack: 1 # Fixes bluryness.
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:26371]:
```suggestion
    wildArmsHack: 1 # Fixes blurriness.
```

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -27639,6 +27651,9 @@ SLPM-66130:
 SLPM-66131:
   name: "Tim Burton's The Nightmare Before Christmas [Premium Pack]"
   region: "NTSC-J"
+  gsHWFixes:
+    wildArmsHack: 1 # Fixes bluryness.
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:27655]:
```suggestion
    wildArmsHack: 1 # Fixes blurriness.
```

File: bin/resources/GameIndex.yaml [hunk 4]
```diff
@@ -29671,6 +29686,9 @@ SLPM-66696:
 SLPM-66697:
   name: "Nightmare Before Christmas, The [CapKore]"
   region: "NTSC-J"
+  gsHWFixes:
+    wildArmsHack: 1 # Fixes bluryness.
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:29690]:
```suggestion
    wildArmsHack: 1 # Fixes blurriness.
```

File: bin/resources/GameIndex.yaml [hunk 5]
```diff
@@ -39527,6 +39545,9 @@ SLUS-20860:
   name: "Nightmare Before Christmas, Tim Burton's The - Oogie's Revenge"
   region: "NTSC-U"
   compat: 5
+  gsHWFixes:
+    wildArmsHack: 1 # Fixes bluryness.
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:39549]:
```suggestion
    wildArmsHack: 1 # Fixes blurriness.
```

File: pcsx2/Docs/GameIndex.md [hunk 6]
```diff
@@ -218,6 +219,9 @@ These values are case-sensitive so take care.  If you incorrectly specify a Game
 *   `VUOverflowHack`
     *   VU Overflow hack to check for possible float overflows (Superman Returns).
 
+*   `SoftwareRendererFMVHack`
+    *   Forces rendering in to software mode during FMVs.
```
Comment (RedDevilus) [inline pcsx2/Docs/GameIndex.md:223]:
```suggestion
    *   Forces rendering into software mode during FMVs.
```

Into in this context will denote change.

File: bin/resources/GameIndex.yaml [hunk 7]
```diff
@@ -14915,6 +14915,9 @@ SLES-53191:
 SLES-53192:
   name: "Nightmare Before Christmas, The - Tim Burton's"
   region: "PAL-M5"
+  gsHWFixes:
+    wildArmsHack: 1 # Fixes bluryness.
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:14919]:
```suggestion
    wildArmsHack: 1 # Fixes blurriness.
```

**Review by RedDevilus — CHANGES_REQUESTED** [review]

**Pull request merged**